packages/synapse-react-client/src/components/TimelinePlot/TimelinePlot.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, StoryObj } from '@storybook/react'
+import TimelinePlot from './TimelinePlot'
+import { rest } from 'msw'
+import { MOCK_REPO_ORIGIN } from '../../utils/functions/getEndpoint'
+import { getHandlersForTableQuery } from '../../mocks/msw/handlers/tableQueryHandlers'
+import { mockTableEntity } from '../../mocks/entity/mockTableEntity'
+import queryResultBundleJson from '../../mocks/query/syn51735464'
+import { getUserProfileHandlers } from '../../mocks/msw/handlers/userProfileHandlers'
+import { ColumnSingleValueFilterOperator } from '@sage-bionetworks/synapse-types'
+
+const meta = {
+  title: 'Components/TimelinePlot',
+  component: TimelinePlot,
+  parameters: { stack: 'mock' },
+} satisfies Meta
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Demo: Story = {
+  args: {
+    sql: `SELECT observationId as "id", observationPhase as "phase", observationSubmitterName as "submitterName", synapseId as "submitterUserId", observationTime as "time", observationTimeUnits as "timeUnits", observationText as "text", observationType as "tag" FROM syn51735464`,
+    searchParams: {
+      resourceId: '9971e47e-976a-4631-8edd-5cae04304b01',
+    },
+    sqlOperator: ColumnSingleValueFilterOperator.EQUAL,
+    defaultSpecies: 'Rattus norvegicus',
```

Comment:
I set the species because I didn't see a way to easily independently mock multiple table query results (one for the event data, and the other that the species selector component executes to find all of the possible species for the resource id)